### ISSUE TITLE
Errors when installing bash install-deps

### ISSUE THREAD

**Issue by 1024064118** [torch/distro#259]
Dear all,
When I tried to run this code to install Torch

git clone https://github.com/torch/distro.git ~/torch --recursive
cd ~/torch; bash install-deps;
./install.sh

I met the error:
Package python-software-properties is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  software-properties-common

E: Package 'python-software-properties' has no installation candidate

But when i tried  sudo apt-get install software-properties-common
it shows this package has been installed:
Reading package lists... Done
Building dependency tree       
Reading state information... Done
software-properties-common is already the newest version (0.96.24.32.3).

Could you please help me to fix it? Thank you very much

**Comment by ZhangYK124** [comment]
same here... Do you have a solution?

**Comment by ddxtanx** [comment]
Yes. Use your favorite text editor and changes lines 178 and 261 of install-deps to
"sudo apt-get install software-properties-common".

As the error message says, python-software-properties has been obsoleted in favour of software-properties-common.

**Comment by Masoud-Karami** [comment]
Hi,
I think it contains four commands. You should either put a semicolon after --recursive or implement them separately:
`git clone https://github.com/torch/distro.git ~/torch --recursive;`
`cd ~/torch; bash install-deps;
./install.sh`
works for me!

**Comment by Robinsssson** [comment]
> Hi, I think it contains four commands. You should either put a semicolon after --recursive or implement them separately: `git clone https://github.com/torch/distro.git ~/torch --recursive;` `cd ~/torch; bash install-deps; ./install.sh` works for me!

It's worked to me, thanks!
